src/shared/components/home/site-form.tsx

```diff
@@ -930,6 +909,83 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
     );
   }
 
+  federatedInstanceSelect(key: InstanceKey) {
+    const id = `create_site_${key}`;
+    const value = this.state.instance_select[key];
+    const selectedInstances = this.state.siteForm[key];
+    return (
+      <div className="col-12 col-md-6">
+        <label className="col-form-label" htmlFor={id}>
+          {i18n.t(key)}
+        </label>
+        <div className="d-flex justify-content-between align-items-center">
+          <input
+            type="text"
+            placeholder="instance.tld"
+            id={id}
+            className="form-control"
+            value={value}
+            onInput={event => this.handleInstanceTextChange(key, event)}
```

Comment:
With inferno, all of the *on* , or anything that deals with event, should use `linkEvent`. Check the other forms for examples.

Reply:
Do you know how to use 'linkEvent'  when passing in parameters other than the event and component? I wanted to use it originally, but I don't know how I'd pass the 'key' variable to the handler in that case.

Reply:
Nvm, it seems that linkEvent can accept things besides events and components as arts.

Reply:
Here's an example: https://github.com/LemmyNet/lemmy-ui/blob/main/src/shared/components/person/settings.tsx#L282